I'm sorry, but an uncaught exception occurred.

While loading <'Image' u'images/backgrounds/bg agradecimentos.png'>:
IOError: Couldn't find file 'images/backgrounds/bg agradecimentos.png'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "renpy/common/_layout/screen_main_menu.rpym", line 28, in script
    python hide:
  File "/home/gabriel/Documentos/Jogos/renpy/renpy/ast.py", line 882, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "/home/gabriel/Documentos/Jogos/renpy/renpy/python.py", line 1913, in py_exec_bytecode
    exec bytecode in globals, locals
  File "renpy/common/_layout/screen_main_menu.rpym", line 28, in <module>
    python hide:
  File "renpy/common/_layout/screen_main_menu.rpym", line 35, in _execute_python_hide
    ui.interact()
  File "/home/gabriel/Documentos/Jogos/renpy/renpy/ui.py", line 289, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "/home/gabriel/Documentos/Jogos/renpy/renpy/display/core.py", line 2662, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, **kwargs)
  File "/home/gabriel/Documentos/Jogos/renpy/renpy/display/core.py", line 3148, in interact_core
    self.draw_screen(root_widget, fullscreen_video, (not fullscreen_video) or video_frame_drawn)
  File "/home/gabriel/Documentos/Jogos/renpy/renpy/display/core.py", line 2065, in draw_screen
    renpy.config.screen_height,
  File "render.pyx", line 487, in renpy.display.render.render_screen
  File "render.pyx", line 235, in renpy.display.render.render
  File "/home/gabriel/Documentos/Jogos/renpy/renpy/display/layout.py", line 719, in render
    surf = render(child, width, height, cst, cat)
  File "render.pyx", line 147, in renpy.display.render.render
  File "render.pyx", line 235, in renpy.display.render.render
  File "/home/gabriel/Documentos/Jogos/renpy/renpy/display/transition.py", line 361, in render
    top = render(self.new_widget, width, height, st, at)
  File "render.pyx", line 147, in renpy.display.render.render
  File "render.pyx", line 235, in renpy.display.render.render
  File "/home/gabriel/Documentos/Jogos/renpy/renpy/display/layout.py", line 719, in render
    surf = render(child, width, height, cst, cat)
  File "render.pyx", line 147, in renpy.display.render.render
  File "render.pyx", line 235, in renpy.display.render.render
  File "/home/gabriel/Documentos/Jogos/renpy/renpy/display/layout.py", line 719, in render
    surf = render(child, width, height, cst, cat)
  File "render.pyx", line 147, in renpy.display.render.render
  File "render.pyx", line 235, in renpy.display.render.render
  File "/home/gabriel/Documentos/Jogos/renpy/renpy/display/screen.py", line 669, in render
    child = renpy.display.render.render(self.child, w, h, st, at)
  File "render.pyx", line 147, in renpy.display.render.render
  File "render.pyx", line 235, in renpy.display.render.render
  File "/home/gabriel/Documentos/Jogos/renpy/renpy/display/layout.py", line 719, in render
    surf = render(child, width, height, cst, cat)
  File "render.pyx", line 147, in renpy.display.render.render
  File "render.pyx", line 235, in renpy.display.render.render
  File "/home/gabriel/Documentos/Jogos/renpy/renpy/display/layout.py", line 1149, in render
    back = render(style.background, bw, bh, st, at)
  File "render.pyx", line 147, in renpy.display.render.render
  File "render.pyx", line 235, in renpy.display.render.render
  File "/home/gabriel/Documentos/Jogos/renpy/renpy/display/im.py", line 580, in render
    return cache.get(self, render=True)
  File "/home/gabriel/Documentos/Jogos/renpy/renpy/display/im.py", line 266, in get
    surf = image.load()
  File "/home/gabriel/Documentos/Jogos/renpy/renpy/display/im.py", line 625, in load
    surf = renpy.display.pgrender.load_image(renpy.loader.load(self.filename), self.filename)
  File "/home/gabriel/Documentos/Jogos/renpy/renpy/loader.py", line 576, in load
    raise IOError("Couldn't find file '%s'." % name)
IOError: Couldn't find file 'images/backgrounds/bg agradecimentos.png'.

Linux-4.15.0-34-generic-x86_64-with-debian-buster-sid
Ren'Py 7.1.0.882
vngamux 1.0
Tue Oct  2 12:28:09 2018
